sdk/communication/Azure.Communication.Common/api/Azure.Communication.Common.netstandard2.0.cs

```diff
@@ -1,23 +1,23 @@
 namespace Azure.Communication
 {
-    public partial class CallingApplication : Azure.Communication.CommunicationIdentifier
+    public partial class CallingApplicationIdentifier : Azure.Communication.CommunicationIdentifier
```

Comment:
Quesiton. Why CallingApplication needs to add suffix "Identifier"?

Reply:
We are adding identifier suffix to all types that implement `CommunicationIdentifier`.
This is both for consistency reasons and because we want to future proof the code :)